docs/development.md

```diff
@@ -106,6 +106,14 @@ For cases like this you can set environment variable `MINK_DRIVER_ARGS_WEBDRIVER
 
 That information will be picked up by testing classes and used for selenium endpoint.
 
+#### How to run the NightwatchJS performance tests
+
+1. You need to install [Yarn](https://yarnpkg.com). Please check installation documentation for it.
```

Comment:
I think we can replace 1. by "Follow the 'Running Nightwatch tests' docs in core/tests/README.md". Then we can also remove 4.

Reply:
I would keep it simple and straight forward.